mount.go

```diff
@@ -53,18 +53,27 @@ func Mount(
 	}
 
 	// Begin the mounting process, which will continue in the background.
+	if config.DebugLogger != nil {
+		config.DebugLogger.Println("Beginning the mounting process")
+	}
 	ready := make(chan error, 1)
 	dev, err := mount(dir, config, ready)
 	if err != nil {
 		return nil, fmt.Errorf("mount: %v", err)
 	}
+	if config.DebugLogger != nil {
+		config.DebugLogger.Println("Completed the mounting process")
```

Comment:
Here we are saying completed the mounting process and below we are saying waiting for mounting process to complete. Isn't is confusing.

Reply:
Done.